chrome.go

```diff
@@ -7,23 +7,22 @@ import (
 )
 
 const (
-	formFieldWaitDelay                  string = "waitDelay"
-	formFieldPaperWidth                 string = "formFieldPaperWidth"
```

Comment:
It looks like this was renamed in a bad find/replace

Reply:
Hi! Yes, indeed, this happened after the renaming to GoLand and requires correction.